packages/astro/src/@types/astro.ts

```diff
@@ -1734,7 +1734,7 @@ export interface AstroUserConfig {
 		 *     prefixDefaultLocale: false,
 		 *		  domains: {
 		 *			  fr: "https://fr.example.com",
-		 *       es: "https://example.es"
+		 *			  es: "https://example.es"
 		 *    },
```

Comment:
it looks like this whole section might be misaligned? although that could be the mix of tabs/spaces.
also noticing that the `i18n` key is missing a closing tag?

Reply:
Oh yeah… it’s all kind of wonky! Thanks — I hadn’t noticed. Will fix and push another commit.

Reply:
Fixed in 0dcb036 🤞